packages/terra-demographics-banner/src/DemographicsBannerDisplay.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { PropTypes } from 'react';
+import { FormattedHTMLMessage, injectIntl, intlShape } from 'react-intl';
+import shortid from 'shortid';
```

Comment:
I'll bring this up in war room tomorrow. I'm not sure how to handle this scenario, and so far this is the best solution I found for generating a unique key for this element.